joomla/joomla-cms
# [HTTPS] Promote the use of secure connections in update channels

#### Summary of Changes

This PR adds a new column "Secure" in updates sites list view to inform the user if the update site use secure https connection or not.

![image](https://cloud.githubusercontent.com/assets/9630530/13724268/6b024bea-e878-11e5-85cd-0164f3ac5476.png)
#### Testing Instructions
1. Install the patch
2. Check if "http:" urls are marked as "No" in the new column and the "https:" url are marked as "Yes". Check the tooltip in the "No" too.
3. Finally check if the ordering of the "Secure" column works in several languages.
